src/styles/utils/_animations.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes tic {
+  from {
+    transform: translateX(-50%) rotate(0deg);
```

Comment:
Why do you need `translateX(-50%)` if it doesn't change?

Reply:
Initially I use `top: 50%; left: 50%; transform: translateX(-50%);` to center hands over the watch.

But when the animation starts it rewrites `transform` to `transform: rotate(0deg);` and the x-offset becomes default so hands are not centralized.

To maintain hands center position along animation I add `translateX(-50%)` to every iteration. And it looks like this:
`transform: translateX(-50%) rotate(0deg);`